table-api/src/main/java/io/deephaven/api/Filter.java

```diff
@@ -0,0 +1,21 @@
+package io.deephaven.api;
+
+public interface Filter {
```

Comment:
gonna stop asking for javadocs, but public api interfaces like these, I'd say it's kind of important, even if just a little blurb / example snippet.